huaweicloud/services/acceptance/scm/data_source_huaweicloud_scm_certificates_test.go

```diff
@@ -37,6 +37,7 @@ func TestAccDatasourceCertificates_basic(t *testing.T) {
 func testAccDatasourceCertificates_basic() string {
 	return `
 data "huaweicloud_scm_certificates" "test" {
+	name = "terrafrom-test"
 }
```

Comment:
```
resource "huaweicloud_scm_certificate" "test" {
  ...
}

data "huaweicloud_scm_certificates" "test" {
  depends_on = [huaweicloud_scm_certificate.test]

  name = huaweicloud_scm_certificate.test.name
}
```
Is it appropriate to not rely on a fixed certificate? (Notes: the name of certificate is not computed)

Reply:
ok, add a preCheck.